packages/react/src/badge/Badge.css.ts

```diff
@@ -0,0 +1,92 @@
+import { theme } from "../styles";
+import {
+  type RecipeVariants,
+  createVar,
+  recipe,
+  style,
+} from "../vanilla-extract";
+
+const subtleBackgroundColorVar = createVar();
+const subtleColorVar = createVar();
+const accentBackgroundColorVar = createVar();
+const accentColorVar = createVar();
+
+export const badge = recipe({
+  base: [
+    {
+      alignItems: "center",
+      fontWeight: "500",
+      justifyContent: "center",
+      px: "8",
+      py: "2",
+      rounded: "sm",
+    },
+    style({
+      textTransform: "capitalize",
```

Comment:
this is a sprinkle prop so we don't need a `style()` call for this - but i don't think we need this at all. consumers are free to use whatever content they want and we shouldn't dictate how it should be capitalized.